src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
-body {
-  margin: 0;
+@import 'card';
+@import 'stars';
+
+:root {
+  --main-accent: #060B35;
+  --secondary: #616070;
```

Comment:
followed the same standard of names...so if its color write `--secondary-color` not just `--secondary`